### DISCUSSION TITLE
Daytime background color

### DISCUSSION THREAD

**Discussion by bdwidgk784** [breezy-weather/breezy-weather#1365] [Ideas]
### Describe your idea

Hello, 

First of all, thank you for everything that you do to maintain this app. We all very much appreciate you. I have used Geometric Weather in the past and loved it. I also like using Breezy Weather. 

However, one thing that I don't like that much in Breezy Weather is the background color when it is a sunny day. The background color in the app is a bright orange color and it is hard to see the weather information like the temperature, humidity, and all of the other information. Especially for people like me with sensitive eyes.

On Geometric Weather, the background was always a sky blue color during the day with only the weather icons changing according to the weather.  The sky blue background color was very eye pleasing and very nice and easy to look at. 

Can you please consider having an option in the settings for that nice sky blue background that was on Geometric Weather? I'm sure that many other people would like to have this option/feature as well. 

### Other details

![1](https://github.com/user-attachments/assets/996c06a2-fa8d-4521-8d4f-2d4e79378a3e)
![2](https://github.com/user-attachments/assets/3d7dcd39-5521-4695-b29f-6693c13f5692)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
It is the exact same orange used in Geometric Weather.

What you show is the "Circular sky" design coming from an outdated version of Geometric Weather that was removed 2 years before the latest commit of Geometric Weather.

There is no way we can bring back such an old design, especially when it was removed by its own author.

However, you can force a blue sky background from the pencil icon on top right of your main screen:
1. Location preferences
2. Weather = Partly cloudy
3. Time = Day

You will have to do this for each location.
